lib/environment/development.js

```diff
@@ -113,6 +114,8 @@ module.exports = class DevelopmentEnvironment {
 
     // Copy the workspace template
     const templateDir = path.join(__dirname, '..', '..', 'resources', 'workspace');
+    const zeroPatchCLIVersion = cliPackageJSON.version.replace(/\d*$/, '0');
```

Comment:
Not sure that such regex is correct if they will use additional labels for pre-release and build metadata
E.g  1.2.3-alpha.1 
You can have a look at `coerce` method of semver package (https://docs.npmjs.com/misc/semver)